### DISCUSSION TITLE
Welcome to RmlUi Discussions!

### DISCUSSION THREAD

**Discussion by mikke89** [mikke89/RmlUi#155] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Put your RmlUi projects on display.
  * Share ideas and thoughts about the library.
  * Show and help each other with integrations.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

For real-time interactions, we also have a [gitter channel](https://gitter.im/RmlUi/community) to reach community members and contributors.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LWSS** [comment]
Hi, LWSS here.

I use RmlUI for one of my personal projects (not public quite yet). Coming mainly from using ImGUI, I wanted something that was more customizable and still ran fast. Webkit-based projects seemed available, but I was unsure how well they would perform. I eventually decided to integrate this project after looking at a few others and it worked out great. The performance is great and it was pretty straight forward to implement the interfaces it requires. The .rml files are similar to simple html and can be picked up quickly if you read the docs. Mikke89, rokups, and others of the community are pleasant to talk to on the gitter, and helped me out if I had a question. 

I actually have very few complaints about the project, one of them is the name though: RmlUI - the I is the main culprit, it's hard to tell if it's an i or an L; I have actually had people call it R-M-(ONE) UI when talking to me, so I just call it RocketUI for slang. If you want a technical one, perhaps there could be more examples, the openGL one does use some deprecated stuff that I didn't know until later. But these are minor complaints I searched for while writing this.

Anyway, in the community I mostly lurk. Once in a while I might come across something minor I want to PR. (Lately I've been thinking about adding a version of ->SetInnerRML() that doesn't attempt to parse it as rml). As a basic-user of this library, I'm quite pleased and I have already recommended it to friends :+1:

**Comment by mikke89** [comment]
Thanks for the kind words LWSS, and a very interesting read! I indeed feel we have a very nice community with many helpful and skilled members. It's been very fun to see this project grow over the last year, and I hope we can retain this if we keep growing. I will at least do my best to make sure the community keeps being a welcoming place :)

Must admit I chuckled a bit at the R-M-One UI. I am so used to the name at this point that I didn't even consider that. I can see it now that you say it, however I don't think it's actually worth it to change the name at this point. Generally, I think most people quickly get accustomed to names whatever they are, so the most important for naming is uniqueness, and I believe we have that.

Yeah, the rendering backend in the samples use very old OpenGL. I'd love to see more backends, it's just a lot of work so I'll have to delegate it to the community if we want to see changes there.

I'll always encourage everyone to do PRs, whether big or small. Though I won't forget you lurkers out there, you have my stamp of approval as well ;) I'm happy as long as you enjoy the library.
